api/server/server.go

```diff
@@ -384,8 +386,14 @@ func createRouter(s *Server) *mux.Router {
 			localFct := fct
 			localMethod := method
 
+			// Define the context that we'll pass around to share info
+			// like the requestID.
+			reqID := stringid.TruncateID(stringid.GenerateNonCryptoID())
+			ctx := context.Background()
+			ctx = context.WithValue(ctx, "requestID", reqID)
```

Comment:
Now that I think about this ID in the context, if we add it to the request header, we don't need to change every handler signature.